tests/IntegrationTests/ItemPropertiesSetupTest.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Model\Command;
+
+/**
+ * @covers ItemPropertiesSetupRequestBuilder
+ * @covers EventsRequestBuilder
+ */
+class ItemPropertiesSetupTest extends IntegrationTestCase
+{
+    /** @test This throws an error in Matej's MGW, but API will return OK */
+    public function issueEventWithUnsetProperties(): void
+    {
+        $response = $this->createMatejInstance()->request()
+            ->events()
+            ->addItemProperty(
+                Command\ItemProperty::create(
+                    'unittest-phpsdk-item-id',
```

Comment:
We're referring to the library as "PHP API client", not SDK - so it can be reflected here as well :).